asv_bench/benchmarks/join_merge.py

```diff
@@ -216,6 +216,9 @@ def time_merge_dataframe_integer_2key(self, sort):
     def time_merge_dataframe_integer_key(self, sort):
         merge(self.df, self.df2, on="key1", sort=sort)
 
+    def time_merge_dataframe_empty(self, sort):
```

Comment:
can you add the reverse as well (e.g. left empty)

Reply:
reverse added here